src/main/java/org/folio/circulation/services/support/RefundCancelAccountCommand.java

```diff
@@ -0,0 +1,18 @@
+package org.folio.circulation.services.support;
+
+import org.folio.circulation.domain.Account;
+import org.folio.circulation.domain.AccountCancelReason;
+import org.folio.circulation.domain.AccountRefundReason;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@Getter
+@AllArgsConstructor
+public final class RefundCancelAccountCommand {
```

Comment:
Does this refund and cancel an account?

If so, what do you think of `RefundAndCancelAccountCommand`?  I think it could be less ambiguous.

Reply:
Yes, renamed.